add-to-pydotorg.py

```diff
@@ -22,6 +22,7 @@
 
 Georg Brandl, March 2014.
 """
+from __future__ import print_function
```

Comment:
Oh boy, I think I would never see you again in this GitHub org :P

Reply:
It looks like the download server, where RMs run the script, currently does have an Ubuntu 3.6.9 installed, so we could consider making the big leap to running under Python 3 :)

Reply:
I could have left it as only valid syntax for Python 2.x if you liked :)